### PR TITLE
S1task6

### PR DESCRIPTION
**Pull request by CosminDarius1**

### CODE REVIEW

File: README.md
Comment (alexjames4) [inline README.md:1]: This is empty? is it needed?

File: index.php [hunk 1]
```diff
@@ -24,13 +23,12 @@
...
+    <header> 
+        <h1>Favorite Movies</h1>
     </header>
-        <section class="card-section">   
-            <?php foreach($allMoviesAsObjects as $movie) {
-                   echo $movie->createCard();
-                   } ?>
+    <section class="card-section">   
+         <?php foreach($allMoviesAsObjects as $movie)
+          {echo $movie->createCard();} ?>
```
Comment (alexjames4) [inline index.php]: { } around the echo is not PSR12 I think?

should be:

foreach($allMoviesAsObjects as $movie) {
          echo $movie->createCard();
}

File: index.php [hunk 2]
```diff
@@ -24,13 +23,12 @@
...
         </section>
```
Comment (alexjames4) [inline index.php]: indentation

File: style.css [hunk 3]
```diff
@@ -3,37 +3,59 @@
...
 }
+html {
+  font-size: 15px;
+}
 body {
   background-color: #e0e0e0;
 }
 header {
-  position: fixed;
+  position: initial;
```
Comment (alexjames4) [inline style.css]: does this do anything?

File: style.css [hunk 4]
```diff
@@ -3,37 +3,59 @@
...
   display: flex;
```
Comment (alexjames4) [inline style.css]: not needed

File: style.css [hunk 5]
```diff
@@ -3,37 +3,59 @@
...
   flex-direction: row;
```
Comment (alexjames4) [inline style.css]: not needed

File: style.css [hunk 6]
```diff
@@ -3,37 +3,59 @@
...
-  justify-content: space-between;
+  justify-content: center;
```
Comment (alexjames4) [inline style.css]: not needed?

File: style.css [hunk 7]
```diff
@@ -3,37 +3,59 @@
...
   width: 100%;
```
Comment (alexjames4) [inline style.css:13]: not needed?

File: style.css [hunk 8]
```diff
@@ -3,37 +3,59 @@
...
   margin-left: 0;
```
Comment (alexjames4) [inline style.css]: not needed?

File: style.css [hunk 9]
```diff
@@ -3,37 +3,59 @@
...
   top: 0;
```
Comment (alexjames4) [inline style.css]: not needed?

File: style.css [hunk 10]
```diff
@@ -3,37 +3,59 @@
...
   left: 0;
```
Comment (alexjames4) [inline style.css]: not needed

File: style.css [hunk 11]
```diff
@@ -3,37 +3,59 @@
...
   z-index: 10;
```
Comment (alexjames4) [inline style.css]: not needed

File: style.css [hunk 12]
```diff
@@ -3,37 +3,59 @@
...
-  padding: 20px;
+  padding: 10px;
+}
+h1 {
```
Comment (alexjames4) [inline style.css:18]: i don't think any of these are needed

File: style.css (hunk 7 above)
Comment (CosminDarius1) [reply style.css:13]: the width?

File: style.css [hunk 13]
```diff
@@ -3,37 +3,59 @@
...
+}
+body {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
 }
 .card-section {
   display: flex;
   flex-wrap: wrap;
   width: 100%;
```
Comment (alexjames4) [inline style.css:29]: not needed

File: style.css [hunk 14]
```diff
@@ -3,37 +3,59 @@
...
-  flex-direction: column;
   max-width: fit-content;
-  margin-left: auto;
-  margin-right: auto;
-  padding: 0 20px;
+  overflow: anywhere;
```
Comment (alexjames4) [inline style.css]: firefox (which i'm using) does not like this.

page seems to display fine so it is not needed

File: style.css [hunk 15]
```diff
@@ -3,37 +3,59 @@
...
+  justify-content: center;
+  font-size: 1rem;
```
Comment (alexjames4) [inline style.css]: 1 rem means it is the same as the default for the page (I think)...is it needed?

File: style.css [hunk 16]
```diff
@@ -3,37 +3,59 @@
...
 }
 .card {
   border: 1px solid black;
   border-radius: 20px;
   margin: 5px;
+  padding: 0 20px;
+  width: 40%;
+  flex-direction: column;
+  display: flex;
+  object-fit: cover;
```
Comment (alexjames4) [inline style.css]: I don't think this is needed

**Review by alexjames4 — CHANGES_REQUESTED** [review]
Looking good, double check my comments whether a property is needed or not.

cheers

**Review by alexjames4 — APPROVED** [review]

**Pull request merged**